src/Raven.Studio/typescript/common/helpers/graph/graphHelper.ts

```diff
@@ -105,6 +105,14 @@ class graphHelper {
         return config;
     }
     
+    static drawErrorLine(ctx: CanvasRenderingContext2D, x: number, y: number, dy: number) {
```

Comment:
this method doesn't draw error line - it draws line

Reply:
done